x509util/name_test.go

```diff
@@ -27,8 +27,9 @@ func Test_newName(t *testing.T) {
 			PostalCode:         []string{"The postalCode"},
 			SerialNumber:       "The serialNumber",
 			CommonName:         "The commonName",
-			ExtraNames: []pkix.AttributeTypeAndValue{
-				{Type: asn1.ObjectIdentifier{1, 2, 840, 113549, 1, 9, 1}, Value: "jane@example.com"},
+			Names: []pkix.AttributeTypeAndValue{
+				{Type: asn1.ObjectIdentifier{2, 5, 4, 3}, Value: "The commonName"},
```

Comment:
This instance of the `The commonName` is ignored, right? It may be clearer to use a different value than the one that's already set in `CommonName`, so that a change in the function will result in a different test outcome.

Reply:
I wanted to "imitate" a real case, Names will have all the attributes present in the subject C, O, OU, CN, ... I will add all of them to make sure the map is correct.

Reply:
Fixed with 4434f23